src/angular-app/languageforge/review-suggest/src/client/app/review/review.component.html

```diff
@@ -59,3 +64,62 @@ <h4 class="left-align">Suggestion </h4>
 </div>
 </div>
 
+
+<div id="settingsModal" class="modal sheet modal-fixed-footer" materialize="modal" [materializeParams]="[{dismissible: true}]" [materializeActions]="settingsModalActions">
+  <div class="modal-content">
+    <div class="row darken-1">
+      <div class="col s6 offset-s6 right-align">
+        <a class="btn-flat darken-1 waves-light waves-effect" (click)="closeModal1()"><i class="material-icons">close</i></a>
+      </div>
+    </div>
+    <div class="row">
+      <h3 class="center-align">Settings</h3>
+    </div>
+    <div class="row">
+    <div class="container">
+      <div class="row">
+        <form action="#">
+          <div class="row">
+          <h5 class="left-align"> Number of Cards</h5>
+          <div class="divider"></div>
+          <p>
+            <input name="group1" type="radio" id="test1" />
```

Comment:
Your name and id names need to be more meaningful. You won't know what they mean a week from now let alone someone else trying to understand your code.

Better yet put the input tag inside the label tag then you don't need ids:
`<label><input type="radio" />25</label>`